libs/features/mas/src/hydrate.js

```diff
@@ -467,6 +467,7 @@ export async function hydrate(fragment, merchCard) {
     merchCard.classList.remove('wide-strip');
     merchCard.classList.remove('thin-strip');
     merchCard.removeAttribute(ANALYTICS_SECTION_ATTR);
+    if (variant === 'plans') merchCard.removeAttribute('segment');
```

Comment:
just noticed, this whole section was accidentally added due to a bad merge.
attributes are removed in cleanup method, could you remove all these duplicating lines pls?

and you can just add `segment` in the array of attributes to be removed

Reply:
Done.